pkg/morph/client/static.go

```diff
@@ -145,20 +155,51 @@ func (s StaticClient) Invoke(prm InvokePrm) error {
 				vubP = &vub
 			}
 
-			_, err = s.client.NotaryInvoke(s.scScriptHash, fee, nonce, vubP, prm.method, prm.args...)
-			return err
+			invokeFunc = func() error {
+				_, err := s.client.NotaryInvoke(s.scScriptHash, fee, nonce, vubP, prm.method, prm.args...)
+				return err
+			}
+		} else {
+			invokeFunc = func() error {
+				return s.client.NotaryInvokeNotAlpha(s.scScriptHash, fee, prm.method, prm.args...)
+			}
+		}
+	} else {
+		invokeFunc = func() error {
+			return s.client.Invoke(
+				s.scScriptHash,
+				prm.await,
+				fee,
+				prm.method,
+				prm.args...,
+			)
 		}
-
-		return s.client.NotaryInvokeNotAlpha(s.scScriptHash, fee, prm.method, prm.args...)
 	}
 
-	return s.client.Invoke(
-		s.scScriptHash,
-		prm.await,
-		fee,
-		prm.method,
-		prm.args...,
-	)
+	expBackoff := backoff.NewExponentialBackOff()
+	err = backoff.RetryNotify(
+		func() error {
+			retries++
+			err = invokeFunc()
+			if err != nil {
+				if errors.Is(err, neorpc.ErrMempoolCapReached) {
+					return err
+				}
+				return backoff.Permanent(err)
+			}
+			return nil
+		},
+		expBackoff,
+		func(err error, d time.Duration) {
+			s.client.logger.Info("retrying due to error", zap.Error(err), zap.Duration("retry-after", d))
+		})
+	if err != nil {
+		s.client.logger.Error("can't Invoke after retries",
```

Comment:
I think we had no messages here before. Usually this is logged in the upper layer because it has more context (like what exactly we're trying to do).